bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -260,4 +264,22 @@ public Map<String, Object> getDorProperties() {
         }
         return customDorProperties;
     }
+
+    @Override
+    @JsonIgnore
+    public String getDorType() {
```

Comment:
we already have a getter i suppose, getDorProperties which returns these values, why do we need these separate getters. we should have only one

Reply:
dorType and dorTemplateRef is specific to form container, I saw **getDorProperties** return a map depending upon component. Isn't it will be good to return specific property as we do for schemaRef and schemaType ?